src/data/ACTIONS/root/DNC.ts

```diff
@@ -277,8 +277,8 @@ export const DNC = ensureActions({
 		onGcd: false,
 		cooldown: 60000,
 		statusesApplied: [
-			'FLOURISHING_SYMMETRY',
-			'FLOURISHING_FLOW',
+			'SILKEN_SYMMETRY', // Pre-6.1 proc statuses for base layer
```

Comment:
This comment reads like the opposite to what it seems to be doing, maybe got these mixed up?

Reply:
Nope, it's correct. The old proc statuses changed their name, so I changed the key in the base layer to reflect the future state, and swapped base Flourish to use the new-key-for-old-status.

In the 6.1 layer I change the name of the old statuses to use the 'Silken' name, officially define the new-status-with-old-names, and update the Flourish action to show the Flourishing-named/keyed statuses again.

Added some more commentary there to clarify.